packages/box/src/styles.ts

```diff
@@ -137,6 +137,9 @@ type LayoutProps = Partial<{
 	flexWrap: ResponsiveProp<'nowrap' | 'wrap' | 'wrap-reverse'>;
 	flexGrow: ResponsiveProp<number>;
 	flexShrink: ResponsiveProp<number>;
+	columnSpan: ResponsiveProp<number>;
```

Comment:
I think this is fine, but I'm wondering if this should be range from `1` to `12`? What are your thoughts @nathsimpson ?

Reply:
I would say no as this property is merely a mapping of the CSS prop, and not tied to a 12 column grid